tests/components/list/items_list.spec.js

```diff
@@ -21,6 +21,15 @@ describe("ItemsList", () => {
     expect(forceUpdateGrid).toHaveBeenCalledTimes(1);
   });
 
+  test("componentDidUpdate will trigger update", () => {
```

Comment:
I would like to suggest testing that this is called when items and selectedItems are changed and not when we force call componentDidUpdate.

Anyway, good catch